draft-packages/empty-state/KaizenDraft/EmptyState/styles.scss

```diff
@@ -150,25 +172,17 @@
 // will enable Zen headings. This block can be removed
 // once these typography styles are the default.
 .zen .heading {
-  @include kz-typography-heading-3;
-
-  .sidebarAndContent & {
-    @include large-sidebar-and-content {
-      @include kz-typography-heading-2;
-    }
-  }
-
-  .contentOnly & {
-    @include large-content-only {
-      @include kz-typography-heading-2;
-    }
-  }
+  font-family: $kz-var-typography-heading-3-font-family;
+  font-size: $kz-var-typography-heading-3-font-size;
+  font-weight: $kz-var-typography-heading-3-font-weight;
+  line-height: $kz-var-typography-heading-3-line-height;
+  letter-spacing: $kz-var-typography-heading-3-letter-spacing;
```

Comment:
Hello reviewer: if you've got more context to the EmptyState component, there seems to be both a Zen and an older variant to the headings. Perhaps the older one can be removed and we can clean this up a bit more? i.e. the `.zen` class might not be needed.

Reply:
Resolved this in slack: https://cultureamp.slack.com/archives/C016N8Y1E10/p1616390689071500

Leaving the .zen variant in place and cleaning later